From 22aa6dc1320bc41e7cfff4d19c8e8229b3d1fd05 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andrius=20=C5=A0tikonas?= <andrius@stikonas.eu>
Date: Sun, 12 Aug 2018 14:26:43 +0100
Subject: [PATCH] [partition] fix build with latest kpmcore.

---
 src/modules/partition/core/PartitionCoreModule.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/modules/partition/core/PartitionCoreModule.cpp b/src/modules/partition/core/PartitionCoreModule.cpp
index f838ab2aa..f6eba9288 100644
--- a/src/modules/partition/core/PartitionCoreModule.cpp
+++ b/src/modules/partition/core/PartitionCoreModule.cpp
@@ -665,7 +665,11 @@ PartitionCoreModule::scanForLVMPVs()
     // Update LVM::pvList
     LvmDevice::scanSystemLVM( physicalDevices );
 
+#ifdef WITH_KPMCOREGT33
+    for ( auto p : LVM::pvList::list() )
+#else
     for ( auto p : LVM::pvList )
+#endif
     {
         m_lvmPVs << p.partition().data();
 

