--- a/src/modules/bootloader/main.py
+++ b/src/modules/bootloader/main.py
@@ -258,7 +258,8 @@
                                "--target=" + efi_target,
                                "--efi-directory=" + efi_directory,
                                "--bootloader-id=" + efi_bootloader_id,
-                               "--force"])
+                               "--force",
+                               "--no-uefi-secure-boot"])
 
         # VFAT is weird, see issue CAL-385
         install_efi_directory_firmware = (vfat_correct_case(
