From 2ea8818d6a0315af611f479ab6190c45f478376d Mon Sep 17 00:00:00 2001
From: Alf Gaida <agaida@siduction.org>
Date: Wed, 29 Mar 2017 16:41:47 +0200
Subject: [PATCH] Introduced help_desc = "foo"

---
 src/modules/testmodule.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/modules/testmodule.py b/src/modules/testmodule.py
index a2d7130..907e704 100755
--- a/src/modules/testmodule.py
+++ b/src/modules/testmodule.py
@@ -67,8 +67,8 @@ def main():
                         help="Dir containing the Python module.")
     parser.add_argument("globalstorage_yaml", nargs="?",
                         help="A yaml file to initialize GlobalStorage.")
-    parser.add_argument("configuration_yaml", nargs="?",
-                        help="A yaml file to initialize the configuration dict.")
+    help_desc = "A yaml file to initialize the configuration dict."
+    parser.add_argument("configuration_yaml", nargs="?", help=help_desc)
     args = parser.parse_args()
 
     print("Testing module in: " + args.moduledir)

