--- a/src/libcalamares/CMakeLists.txt
+++ b/src/libcalamares/CMakeLists.txt
@@ -92,8 +92,8 @@
 
 install( TARGETS calamares
     EXPORT CalamaresLibraryDepends
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/calamares
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/calamares
 )
 
 # Install header files
